assembl/models/preferences.py

```diff
@@ -546,6 +546,17 @@ def get_preference_data_list(self):
             "default": True,
         },
 
+        # Show different messages in case of a failed registration
+        {
+            "id": "generic_auth_errors",
+            "name": _("Generic Auth Errors"),
+            "value_type": "bool",
+            "description": _("Display a generic error message in case of bad authentication"),
```

Comment:
This isn't just for auth, it's being used for auth for now. Can set this to be `generic_errors` and change the description to generalize all errors.